### ISSUE TITLE
[bug] - Add to cart button is not working ( WooCommerce ). 

### ISSUE THREAD

**Issue by protob** [vuefront/vuefront#15]
My setup is:
Wordpress 5.3
WooCommerce 3.8.0 
VueFront CMS Connect App 1.2.0

Add to cart button is not working, Actuallly, the button text is "add to bug". After clicking the button, it shows "Internal server error" notificaiton.  I tested it with WooCommerce both on localhost and remote server.
![screenshot-localhost_3000-2019 11 27-10_21_15](https://user-images.githubusercontent.com/24397445/69716042-8f03f300-1109-11ea-89db-712bbb8b2e96.png)

**Comment by marcjull** [comment]
This has been happening to me in several attempts to install woommerce, at the next step just after Add to Cart, the purchase conclusion step, when the endpoint should work ("order-sent" or something like this).

The theme was mostly Ocean WP, once with Storefront, which is woocommerce's official plugin, and even with one of those WP Twenty themes, when I tried a procedure suggested by Woocommerce for identifying identify the problem. I was also using Elementor.

**Comment by Ecoent** [comment]
Hi @marcjull, @protob, 

Happy New Year to everyone here!

Did you use your own themes? 
in other words, i want to know whether we can use a 3rd party theme and still use VueFront app extension on Wordpress / Woocommerce. 
We like to stick to the current theme, consistency in look and feel, and then may be in future switch to a new theme. We have a marketplace on woo, so a lot of work to switch themes. :-(

It will be great if existing theme works with VueFront. Nothing is mentioned clearly in the docs or setup, hence the question.

thanks,
Eco

**Comment by protob** [comment]
Hi, My experience with VueFront is limited to testing its possibilities with defualt theme, i've never used it in real wolrd project with or with any custom theme.
My requirement was to have highly cuustomizable,  Vue.js based, WooCommerce integration without the need of extra infrastructure.
I ended with building my own solution (still under development) , based on Wp GraphQL Woocommerce plugin https://github.com/wp-graphql/wp-graphql-woocommerce and Nuxt.js

**Comment by Ecoent** [comment]
Thanks a ton for your reply @protob 
i am testing the VueFront things on a dev but for a real project. The docs are really confusing, i also found a new version here than the one on Wordpress.
I will check that GraphQL out, seen that earlier, thanks for sharing that link.

**Comment by protob** [comment]
Yes, the plugin exposes self-documenting GraphQL endpoint for WooCommerce store. I used Apollo client to communicate with API.
